src/cpp/ext/otel/otel_plugin.h

```diff
@@ -398,6 +400,8 @@ class OpenTelemetryPlugin : public grpc_core::StatsPlugin {
   // Instruments for per-call metrics.
   ClientMetrics client_;
   ServerMetrics server_;
+  std::vector<grpc_core::ClientCallTracer::CallAttemptTracer::OptionalLabelKey>
```

Comment:
I think this can be `std::bitset<OptionalLabelKey::kSize>`, using the same approach that we use for optional labels for the NPC metrics.

Reply:
I agree. I was trying to avoid conversions between the enum and size_t, but that led to use of a map over an std::array. The benefits of an std::array where we avoid per-call allocations and get contiguous memory access vastly outweigh the conversion downside.